### PR TITLE
added beginnings of a troubleshooting guide for the documentation

### PR DESCRIPTION
**Pull request by adampitchie**
Since there seems to be some common issues that users face, it seemed prudent to add a guide to reduce repetitive issues for contributors.

### CODE REVIEW

**Comment by adampitchie** [comment]
What other common issues do contributors see? Link to them and I'll add them.

**Comment by phatblat** [comment]
Something simple to try before deleting files is to close and reopen the Xcode workspace.

screen shot 2016-05-01 at 6 42 05 pm

I just got these errors by opening the workspace after a fresh clone of a [project with an ignored `Pods` folder](https://github.com/phatblat/OutletActionAssertion). I ran `pod install` while it was still open but Xcode wasn't even trying to build Quick until I closed/reopened the workspace.

File: Documentation/en-us/Troubleshooting.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Common Installation Issues
+
+Here are solutions to common issues that come up when using the framework.
+
+## No such module 'Quick'
+
+- Delete the _entire_ `~/Library/Developer/Xcode/DerivedData` direction, which includes `ModuleCache`.
+- Explicitly build (`Cmd+B`) the `Quick`, `Nimble`, and `Pods-ProjectNameTests` targets after enabled their schemes from the Manage Schemes dialog.
```
Comment (madson) [inline Documentation/en-us/Troubleshooting.md]: Thanks! It worked for me, but I don't need to build `Quick` and `Nimble` separately. I've just build `Pods-ProjectNameTests` after enabled that scheme.

**Comment by takecian** [comment]
@adampitchie , could you add @phatblat 's comment? After that, i will merge this PR.

**Comment by adampitchie** [comment]
@takecian Done!

**Comment by takecian** [comment]
Merged. Thanks @adampitchie !

**Pull request merged**